docs/source/installation.rst

```diff
@@ -43,10 +44,10 @@ Mac OS X
 Linux
     Use your package manager to install ``numpy`` and ``scipy``
         Ubuntu: ``sudo apt-get install python-numpy python-scipy``
-    Run ``pip install pint``
+    Run ``pip install pint`` (for system python installs, use ``sudo pip``)
```

Comment:
You have to use sudo with apt-get. There is no option there. pip is a different story. You can install in your system with sudo pip, or you can install in your home directory with --user. I'm not sure what we should recommend. If we suggest --user for gpkit though people won't have the issue with the $HOME/.gpkit having root permissions that they have to fix.